# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/GLImages.h
# Commit: c333871e1c59
# Full Hash: c333871e1c59abe6daf30c396e5e0a09591ac440
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2022-01-19 09:34:35
# Regressor Bug: 1640607
# File Overlap Count: 1
# Description:
#   Bug 1736793 - Add SurfaceTexture support to GLBlitHelper::BlitSdToFramebuffer() r=jgilbert,gfx-reviewers
#   
#   Blit support of android SurfaceTexture was removed by Bug 1640607.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D128992
# ==============================================================================

diff -r bb40a646b246 -r c333871e1c59 gfx/layers/GLImages.h
--- a/gfx/layers/GLImages.h	Wed Jan 19 02:03:31 2022 +0000
+++ b/gfx/layers/GLImages.h	Wed Jan 19 02:13:48 2022 +0000
@@ -59,6 +59,8 @@
 
   SurfaceTextureImage* AsSurfaceTextureImage() override { return this; }
 
+  Maybe<SurfaceDescriptor> GetDesc() override;
+
   void RegisterSetCurrentCallback(UniquePtr<SetCurrentCallback> aCallback) {
     mSetCurrentCallback = std::move(aCallback);
   }
